bin/warmup_stats

```diff
@@ -144,16 +144,14 @@ def create_arg_parser():
                         dest='debug_level',
                         help='Debug level used by logger. Must be one of: '
                              'DEBUG, INFO, WARN, DEBUG, CRITICAL, ERROR')
+    csv_only_msg = ('Only needed if input files are in CSV format. Note that CSV '
```

Comment:
Hah. Oops. I meant, can it be displayed only once in the help text?

Isn't there a `prolog` method to add text to the start?

Reply:
It can, but I think it needs to also go here, since some people will skip over the details above. Let me push a commit and see what you think.